fe/src/project/Account/areas/Accounts/components/SignIn/SignInForm.js

```diff
@@ -102,7 +105,9 @@ SignInForm.propTypes = {
     registerLabel: PropTypes.string.isRequired,
     developersEmail: PropTypes.string.isRequired,
     contactText: PropTypes.string.isRequired,
-    registerUrl: PropTypes.string.isRequired
+    registerUrl: PropTypes.string.isRequired,
+    isInvalidPassword: PropTypes.bool,
```

Comment:
**suggestion:** There should be only one prop on the FE "ErrorMessage". There is no need to create a separate flag isInvalidPassword for it and make it so specific. Just display the error message if the ErrorMessage prop is set.